apis_core/apis_relations/autocomplete_light_registry.py

```diff
@@ -1,257 +0,0 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
-import operator
-import re
-from functools import reduce
-import importlib
-
-import autocomplete_light.shortcuts as al
-import dateutil.parser
-import requests
-from django.db.models import Q
-from django.conf import settings
-
-from apis_core.apis_entities.models import Place, Person, Institution, Event, Work
-from apis_core.apis_metainfo.models import Uri
-from apis_core.apis_relations.models import PersonPerson, PersonPlace, PersonInstitution
-
-path_ac_settings = getattr(settings, "APIS_AUTOCOMPLETE_SETTINGS", False)
-if path_ac_settings:
-    ac_settings = importlib.import_module(path_ac_settings)
-    ac_settings = getattr(ac_settings, "autocomp_settings")
-else:
-    from apis_core.default_settings.NER_settings import autocomp_settings as ac_settings
-
-
-class StanbolAutocompleteBase(al.AutocompleteListTemplate):
```

Comment:
That function provided the autocomplete suggestions for entities derived from our stanbol instance. Given that we dont have a fixed set of entity types anymore this does not make much sense anymore. However, we want to indlude a way in apis-rdf-core to hook additional autocomplete sources to the entities autocompletes. E.g. a list of tuples in the settings with the entity and the function that recieves the query string and returns a configurable amount of suggestions in a defined format.